src/main/kotlin/no/nb/mlt/wls/order/model/Order.kt

```diff
@@ -4,12 +4,18 @@ import io.swagger.v3.oas.annotations.media.Schema
 import io.swagger.v3.oas.annotations.media.Schema.AccessMode.READ_ONLY
 import no.nb.mlt.wls.core.data.HostName
 import no.nb.mlt.wls.core.data.Owner
+import org.bson.types.ObjectId
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.index.CompoundIndex
 import org.springframework.data.mongodb.core.mapping.Document
 
+@CompoundIndex(unique = true, def = "{'hostName':1,'hostOrderId':1}")
```

Comment:
Should also have it for products, I think. Eg. hostName and productId, since they also should be unique

Reply:
This change should be easy to implement in the DB and the code. I can include it in this PR if desirable